doc/en/components/grids/pivot-grid/overview.md

```diff
@@ -2,13 +2,368 @@
+        {
+            MemberName = "Product",
+            Enabled = true,
+            Name = "pivotDimension1"
+        });
+    pivotConfiguration.Columns.Add(new IgbPivotDimension()
+        {
+            MemberName = "Country",
+            Enabled = true,
+            Name = "pivotDimension2"
+        });
+    pivotConfiguration.Values.Add(new IgbPivotValue()
+        {
+            Member = "UnitsSold",
+            Name = "pivotValue1",
+            Enabled = true,
+            Aggregate = new IgbPivotAggregator() { Key = "sum", AggregatorName = PivotAggregationType.SUM, Label = "Sum" }
+        });
+}
+```
+
+This configuration defines 1 row, 1 column and 1 aggregation that sums the values of each dimension groups.
+The members match fields available in the provided data source:
+
+```typescript
+public data = [
+[
+    {
+        Product: {
+            Name: 'Clothing',
+            UnitPrice: '12.814860936633712'
+        },
+        Seller: {
+            Name: 'Stanley Brooker',
+            City: 'Seattle'
+        },
+        Date: '2007-01-01T00:00:00',
+        Value: '94.2652032683907',
+        NumberOfUnits: '282'
+    },
+    //...
+];
+```
+
+```
+public PivotSalesData()
+{
+    this.Add(new PivotSalesDataItem()
+    {
+        Country = @"UK",
+        Product = @"Vermont",
+        UnitsSold = @"501",
+        ManufacturingPrice = 15,
+        SalePrice = 23,
+        GrossSales = 26440,
+        Discounts = double.NaN,
+        Sales = 26440,
+        COGS = 16185,
+        Profit = 11255,
+        Date = @"1/1/20",
+        MonthName = @"January",
+        Year = @"2020"
+    });
+```
+
+
+Resulting in the following view, which groups the Product Categories unique columns, Sellers Countries in unique rows and displays the related aggregations for the number of units in the related cells:
+
+<code-view style="height: 530px"
+           data-demos-base-url="{environment:demosBaseUrl}"
+           iframe-src="{environment:demosBaseUrl}/{PivotGridSample}-basic"
+           alt="{Platform} Pivot Grid Basic Example">
+</code-view>
+
+
+## Known Issues and Limitations
+
+|Limitation|Description|
+|--- |--- |
+| Setting columns declaratively is not supported. | The Pivot grid generates its columns based on the `Columns` configuration, so setting them declaratively, like in the base grid, is not supported. Such columns are disregarded. |
+| Setting duplicate `MemberName` or `Member` property values for dimensions/values. | `MemberName`/`Member` should be unique for each dimension/value. Duplication may result in loss of data from the final result. |
+| Row Selection is only supported in **Single** mode. | Multiple selection is currently not supported. |
+| Merging the dimension members is case sensitive| The pivot grid creates groups and merges the same (case sensitive) values. But the dimensions provide `MemberFunction` and this can be changed there, the result of the `MemberFunction` are compared and used as display value.|
```

Comment:
For Angular only regarding `MemberFunction` as well (if the table can work that way anyway)